hdeps/tests/__init__.py

```diff
@@ -1,7 +1,14 @@
 from .cache import SimpleCacheTest
 from .markers import EnvironmentMarkersTest
+from .requirements import RequirementsTest
+from .resolution import ResolutionTest
+
+# from .session import LiveSessionTest
```

Comment:
This tests exists but is the only one that actually needs to hit the network.  I don't want to just call `get_*_session()` and throw away the result, because that gives a false sense of security.  So I'm leaving the lines uncovered for now as a reminder for future-me...